tests/test_main.py

```diff
@@ -363,11 +410,18 @@ def test_distance_class_le_number(kilometers, kilometers2, result):
     [
         50,
         100,
-        300
+        300,
+        100.0,
+        12.0
     ]
 )
 def test_distance_class_len(kilometers):
     distance = Distance(kilometers)
-    assert len(distance) == distance.km, (
-        f"'len()' for instance should return instance.km"
-    )
+    if isinstance(kilometers, int):
+        assert len(distance) == distance.km, (
+            f"'len()' for instance should return instance.km"
+        )
+    else:
+        assert kilometers == distance.km, (
+            f"'len()' for instance of float type should return instance.km"
```

Comment:
I believe this comment is not really correct.
`len` for instance of float type will be an error. Smth like `len(2.3)` -> error.
I believe the good message is `"'len()' for instance of Distance should return instance.km"`